initfiles/componentfiles/configxml/@temp/esp_service_DynamicESDL.xsl

```diff
@@ -100,7 +100,7 @@
                         <xsl:message terminate="yes">ESP NetAddress is undefined!</xsl:message>
                     </xsl:if>
                     <xsl:variable name="wsloggingUrl"><xsl:text>http://</xsl:text><xsl:value-of select="$espNetAddress"/><xsl:text>:</xsl:text><xsl:value-of select="$espPort"/></xsl:variable>
-                        <LogAgent name="{$agentName}" type="LogAgent" services="GetTransactionSeed,UpdateLog" plugin="espserverloggingagent">
+                        <LogAgent name="{$agentName}" type="LogAgent" services="GetTransactionSeed,UpdateLog,GetTransactionID" plugin="espserverloggingagent">
```

Comment:
The @services attribute includes the functionalities this Logging Agent is designed for.